jax/_src/numpy/lax_numpy.py

```diff
@@ -1464,6 +1464,17 @@ def in1d(ar1, ar2, assume_unique=False, invert=False):
   else:
     return (ar1[:, None] == ar2).any(-1)
 
+@_wraps(np.setdiff1d, lax_description="""
+In the JAX version, the `assume_unique` argument is not referenced.
+""")
+def setdiff1d(ar1, ar2, assume_unique=False):
+
+  ar1 = unique(ar1)
+  ar2 = unique(ar2)
```

Comment:
Could we skip the `unique()` call when `assume_uniqe` is True?

Reply:
Sure, I've now understood that in1d actually assumes that they are unique :)

Reply:
I've tried to make this work, but due to `jnp.in1d`'s behavior it will not work. Although working for most of the tests, there is no guarantee it will match `np.in1d` with `assume_unique=True` and this is the default way that `np.setdiff1d()` calls it.

Reply:
Ah, good point. I hadn't thought of that.